content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,191 @@
+
+2. Ensure you have deployed the [Datadog Lambda Forwarder][2], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.66.0+. You may need to [update your Forwarder][5].
+
+   Take note of your Forwarder's ARN.
+
+3. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
+   {{< img src="serverless/step_functions/aws_log.png" alt="AWS UI, Logging section, showing log level set to ALL." style="width:100%;" >}}
+
+4. Add the following to your `serverless.yml`:
+
+    ```yaml
+    custom:
+      datadog:
+        site: <DATADOG_SITE>
+        apiKeySecretArn: <DATADOG_API_KEY_SECRET_ARN>
+        forwarderArn: <FORWARDER_ARN>
+          enableStepFunctionTracing: true
+    ```
+    - Replace `<DATADOG_SITE>` with {{< region-param key="dd_site" code="true" >}} (ensure the correct SITE is selected on the right).
+    - Replace `<DATADOG_API_KEY_SECRET_ARN>` with the ARN of the AWS secret where your [Datadog API key][3] is securely stored. The key needs to be stored as a plaintext string (not a JSON blob). The `secretsmanager:GetSecretValue` permission is required. For quick testing, you can instead use `apiKey` and set the Datadog API key in plaintext.
+    - Replace `<FORWARDER_ARN>` with the ARN of your Datadog Lambda Forwarder, as noted previously.
+
+    For additional settings, see [Datadog Serverless Framework Plugin - Configuration parameters][7].
+
+5. For Node.JS and Python runtimes, set `mergeStepFunctionAndLambdaTraces:true` in your `serverless.yaml` file. This links your Step Function traces with Lambda traces. If you have not instrumented your Lambda functions to send traces, you can [follow the steps to add the Lambda layer for your preferred runtime][8].
+
+[1]: https://docs.datadoghq.com/serverless/libraries_integrations/plugin/
+[2]: /logs/guide/forwarder/
+[3]: https://app.datadoghq.com/organization-settings/api-keys
+[4]: https://www.serverless.com/
+[5]: /logs/guide/forwarder/?tab=cloudformation#upgrade-to-a-new-version
+[6]: logs/guide/forwarder/?tab=cloudformation#installation
+[7]: serverless/libraries_integrations/plugin/#configuration-parameters
+[8]: /serverless/installation/#installation-instructions
+{{% /tab %}}
+{{% tab "Datadog Serverless CLI" %}}
+1. If you have not already, install the [Datadog Serverless CLI][1] v2.18.0+.
+
+   ```shell
+   npm install -g @datadog/datadog-ci
+   ```
+2. Ensure you have deployed the [Datadog Lambda Forwarder][2], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.66.0+. You may need to [update your Forwarder][3].
+
+   Take note of your Forwarder's ARN.
+3. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
```

Comment:
Optional: I've seen actions bolded in other parts of the docs, example: Click **edit**, Select **option**
but there is consistency here so making an optional mention.

Reply:
yeah... cross-docs consistency for this type of thing is probs a style council issue (there was actually a discussion about it yesterday in slack, incidentally) — but for now i'm going to prioritize section-specific consistency.